src/unity/python/README.rst

```diff
@@ -0,0 +1,35 @@
+Turi Create
+===========
+
+Turi Create simplifies the development of custom machine learning
+models. You don’t have to be a machine learning expert to add
+recommendations, object detection, image classification, image
+similarity or activity classification to your app.
+
+Supported Platforms Turi Create supports:
```

Comment:
Change this to just say `Supported Platforms` - the rest is redundant since this whole document is about Turi Create.

Reply:
Oh, it's a typo, probably from the md-->rst conversion.
There should be a \<newline> there, as in:

Supported Platforms 
--------------------

Turi Create supports:
mac ...